PCS for another operation is in progress error (#1)
redkubes/redkubes.github.io: docs/ki-cli.md

https://issuetracker.google.com/issues/171493249

Maybe they will start to see the importance of this after getting more feedback ;)

### 4. It can't deploy when another operation is in progress

**Problem**: The `otomi apply` or `otomi sync` command fails with the following error:

```
Error: UPGRADE FAILED: another operation (install/upgrade/rollback) is in progress
```

**Cause**: A given helm release is in `pending-upgrade` state, e.g.:

```
NAME      NAMESPACE    REVISION    UPDATED                 STATUS     CHART      APP VERSION
keycloak    keycloak    3        2021-03-22 13:50:22.5069506 +0000 UTC  pending-upgrade keycloak-8.2.2 10.0.0
```

**Solution**: Find helm release that is in the `pending` state:

```
helm list -a -A | grep -i pending
```

> If there is helm release in `pending` state and it has more than one revision, then rollback to the previous revision:

Reviewer: ```suggestion
If there is an helm release in the state `pending` AND it has more than one revision, then rollback to the previous revision:
```

Author: done

```
helm -n <namespace> rollback <release-name> <previous revision number>
```

> Otherwise, if there is helm realease in `pending` state and it has only one revision, then remove that release:

Reviewer: ```suggestion
Otherwise, if there is an helm release in the state `pending` AND it has only one revision, then remove that release:
```

keycloak    keycloak    3        2021-03-22 13:50:22.5069506 +0000 UTC  pending-upgrade keycloak-8.2.2 10.0.0
```

**Solution**: Find helm release that is in the `pending-upgrade` state:

```
helm list -a -A | grep -i pending
```

If there is a helm release in the `pending` state AND it has more than one revision, then rollback to the previous revision:

```
helm -n <namespace> rollback <release-name> <previous revision number>
```

> Otherwise, if there is an helm release in the state `pending` AND it has only one revision, then remove that release:

Reviewer 2: ```suggestion
Otherwise, if there is a helm release in the state `pending` AND it has only one revision, then remove that release:
```
